packages/vc-handler-ld-local/src/__tests__/context.loader.test.ts

```diff
@@ -0,0 +1,39 @@
+import { ContextDoc, LdDefaultContexts } from '@veramo/credential-ld'
+
+import { LdContextLoader } from '../ld-context-loader'
+
+describe('credential-ld context loader', () => {
+  const customContext: Record<string, ContextDoc> = {
+    'https://example.com/custom/context': {
+      '@context': {
+        '@version': 1.1,
+        id: '@id',
+        type: '@type',
+        nothing: 'https://example.com/nothing',
+      },
+    },
+  }
+
+  it('loads custom context from record', async () => {
```

Comment:
I would start statement with a 'should' since it's a test and there is no guarantee that it will pass.

Reply:
yes, fixed, although this came from the original code